src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/DisplayNameProvider.cs

```diff
@@ -24,5 +24,11 @@ internal abstract class DisplayNameProvider
         /// <see cref="TryGetLogicalName(DName, out DName)"/>, with the newDisplayName output populated by the first arg. 
         /// </summary>
         public abstract bool TryRemapLogicalAndDisplayNames(DName displayName, out DName logicalName, out DName newDisplayName);
+
+        public abstract IEnumerable<KeyValuePair<DName, DName>> LogicalToDisplayPairs { get; }
+
+        internal DisplayNameProvider()

```

Comment:
We can use `private protected` instead of `internal` now. https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/private-protected

Reply:
got it